lib/msal-common/src/account/AccountInfo.ts

```diff
@@ -21,5 +22,5 @@ export type AccountInfo = {
     username: string;
     localAccountId: string;
     name?: string;
-    idTokenClaims?: object;
+    idTokenClaims?: TokenClaims & { [key: string]: string | number | string[] | undefined | unknown };
```

Comment:
`object` is also valid.

```suggestion
    idTokenClaims?: TokenClaims & { [key: string]: string | number | string[] | object | undefined | unknown };
```